### DISCUSSION TITLE
is there way set time delay slideshow?

### DISCUSSION THREAD

**Discussion by nquantum** [d2phap/ImageGlass#722] [General] [answered]
### System information:
 - Windows OS version: 10 1909
 - ImageGlass version: 7.5.1.1

### Questions
when in slideshow mode. is there a way to set time delay between pictures? instead of default 5sec? because I not found any relate in setting

**Comment by d2phap** [comment, marked as answer]
Hi @nquantum 
Yes you can change it in Settings > Image > Slideshow :)

![image](https://user-images.githubusercontent.com/3154213/77287766-5075f200-6d11-11ea-8de9-11c357ffcc14.png)
